Week-3/CSS-BASICS/q1.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Auto Purchase Offer and Loan Comparison</title>
+    <style>
+
+        body{
+            font-family:  Roboto, Helvetica,sans-serif;
+        }
+
+        h1{
+            font-size:16px;
+         color: #444444;
+         text-align: center;
+        }
+
+        .container{
+            text-align: center;
+         margin: 5px;
+         font-size:  13px;
+
+        }
+
+        .offer{
+            display: inline-block;
+            width: 400px;
+         margin: 4px;
+         text-align: center;
+         border: 1px solid #cccccc;
+        }
+
+        .offerTitle{
+            font: 14px;
+         font-weight: bold;
+         font-size: 14px;
+         color: #ffffff;
+         background-color: #00b3b3;
+         padding: 15px;
+         margin: 0;
+
+        }
+
+        .offerContents{
+            padding:20px;
+         background-color: #fafafa;
+
+        }
+
+         .table {
+            width: 100%;
+            background-color: #ffffff;
+            border-spacing: 0;
+         }
+
+         tr {
```

Comment:
Use CSS classes for `tr `and `td `elements instead of direct styling.

```
.offer-table tr {
  /* ... styles for tr ... */
}
.offer-table td {
  /* ... styles for td ... */
}

```